README.md

```diff
@@ -219,6 +219,34 @@ $ hjt golang
        2011-01     2012-01     2013-01     2014-01     2015-01     2016-01     2017-01     2018-01     2019-01
 ```
 
+#### AI/ML trends
+
+```bash
+
+$ hjt 'ai' + 'artifical intelligence' + 'ml' + 'machine learning'
```

Comment:
'ai' and 'ml' is not good matcher, because for example 'ai', will also match 'maid'.
maybe 
```bash
hjt ' ai ' + ' ml ' + 'artifical intelligence' + 'machine learning' + 'tenserflow'
```